README.md

```diff
@@ -4,7 +4,27 @@
 [![Chat](https://img.shields.io/static/v1.svg?label=chat&message=mattermost&color=145dbf)](https://mattermost.eclipse.org/eclipse/channels/eclipse-codewind)
 
 # Codewind for VS Code
-Create and develop cloud-native, containerized web applications from VS Code.
+Codewind for VS Code provides tools to help build cloud-native, containerized applications from VS Code, regardless of which language you use. Rapidly create an application from a template project and launch, update, test, and debug in Docker containers on the desktop.
+
+## Why use Codewind?
+- Work on your desktop then run and debug on Kubernetes.
```

Comment:
"debug on Kubernetes" is misleading because we don't support debugging while running on Kube yet, that is a ways away still.

We _do_ support debugging applications in Docker containers, which is worth mentioning (it is mentioned in the Features list below).
We _will_ support _deploying_ from desktop to Kubernetes, but that has not hit production yet.